ui/app/AppLayouts/Wallet/panels/TokenSelectorPanel.qml

```diff
@@ -0,0 +1,303 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1
+import StatusQ.Popups.Dialog 0.1
+
+import AppLayouts.Wallet.views 1.0
+import shared.controls 1.0
+import utils 1.0
+
+import SortFilterProxyModel 0.2
+
+Control {
+    id: root
+
+    property alias assetsModel: assetsSfpm.sourceModel
```

Comment:
Would be nice to describe the model structure for the `assetsModel` too

Reply:
Esp. because for the SwapModal, I'll most likely have to use a different adaptor

Reply:
I forgot, definitely should be there